### PR TITLE
added important note 

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8123

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Heidilohr** [comment]
@e0i I'm afraid I don't handle the CSP docs anymore. If you get in touch with Daniel Simpson (dansimp@microsoft.com), he'll be able to tell you who the current writer who approves CSP-related pull requests is.

File: windows/client-management/mdm/policy-csp-servicecontrolmanager.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ If you enable this policy setting, built-in system services hosted in svchost.ex
 
 This includes a policy requiring all binaries loaded in these processes to be signed by Microsoft, as well as a policy disallowing dynamically-generated code.  
 
+> [!IMPORTANT]
+> Enabling of this policy could cause compatibility issues with third-party software that uses svchost.exe processes (for example, third-party antivirus software).
```
Comment (illfated) [inline windows/client-management/mdm/policy-csp-servicecontrolmanager.md:82]: I wonder if this phrasing is better without "of":
```suggestion
> Enabling this policy could cause compatibility issues with third-party software that uses svchost.exe processes (for example, third-party antivirus software).
```
To use "of" here, it should probably be phrased like "The enabling of ..."

**Review by illfated — APPROVED** [review]
Looks good to me. Thank you for this update.

**Comment by ManikaDhiman** [comment]
Reached out to the policy owner to review the note; awaiting response.

**Comment by ManikaDhiman** [comment]
Important note approved by policy owner; merging the PR. Thanks!

**Pull request merged**